fv/service_loop_prevention_test.go

```diff
@@ -306,6 +306,13 @@ var _ = infrastructure.DatastoreDescribe("service loop prevention; with 2 nodes"
 			cfg.Spec.ServiceLoopPrevention = "Disabled"
 		})
 
+		// Expect to see empty cali-cidr-block chains.  (Allowing time for a Felix
+		// restart.)  This ensures that the cali-cidr-block chain has been cleared
+		// before we try a test ping.
+		for _, felix := range felixes {
+			Eventually(getCIDRBlockRules(felix, "iptables-save"), "8s", "0.5s").Should(BeEmpty())
```

Comment:
qq:  Should this include the iptables6-save sim. to the inverse checks above?

Reply:
Actually .... I don't think these checks are correct.  You need to pass in a function to these....

so something like:

`Eventually(func()  []Rules {return getCIDRBlockRules(felix, "iptables-save")})...`

Reply:
That should be what `getCIDRBlockRules` does.  I'll just check...

Reply:
It does
I deleted my comment ... but you got there too soon :-)

Reply:
I didn't include the v6 check because it isn't strictly needed for the v4 ping to behave as we expect.